app/views/pipelines/_card.html.erb

```diff
@@ -1,19 +1,32 @@
 <% jobs_path ||= ''
    run_sample_extraction_path ||= ''
-   run_full_extraction_path ||= '' %>
+   run_full_extraction_path ||= ''
+   type = definition.class.name.gsub(/Definition$/, '').downcase
+   position = type == 'extraction' ? 'first' : 'last' %>
 
 <div class="card">
   <div class='card-body'>
-    <h5 class="card-title"><%= definition.send("#{type}_definition").name %></h5>
+    <h5 class="card-title"><%= definition.name %></h5>
```

Comment:
Ah I see this is our own controlled classes. Should use BEM then :)

Reply:
Do you think? I don't want to recreate the bootstrap cards. I just want to make the bootstrap cards fit the design.

Reply:
That's fair, I forgot a bit how bootstrap / foundation works.  All good!